asciidoc-confluence-publisher-client/src/it/java/org/sahli/asciidoc/confluence/publisher/client/ConfluencePublisherIntegrationTest.java

```diff
@@ -104,6 +104,30 @@ public void publish_singlePageAndReplaceAncestorPublishingStrategy_pageIsUpdated
                 .body("results[1].title", is("attachmentTwo.txt"));
     }
 
+    @Test
+    public void publish_sameAttachmentsPublishedMultipleTimes_doesNotAddTheSameAttachments() {
```

Comment:
Is this test really testing the optimization when uploading attachments? Would it realize about the same attachment being uploaded twice, with the second one overwriting the first one with the same name? I don't think so.  I understand this test as being unrelated to the attachment optimization. If so, I would remove it from this pull request.

Reply:
Ok. Do you have any thoughts on what integration tests should be added?